cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -23,6 +24,20 @@
 ], scope="module")
 
 
+@pytest.fixture(scope="session")
```

Comment:
This fixture enables websocket role for whole session, maybe it's better to use "module" scope?